assets/docs/CallmeMehdi/models/AraBERT/1.md

```diff
@@ -0,0 +1,23 @@
+# Module CallmeMehdi/arabert/1
+
+AraBERT is an Arabic pretrained lanaguage model based on Google's BERT architechture.
+
+<!-- asset-path: https://firebasestorage.googleapis.com/v0/b/gsoc-tf.appspot.com/o/arabert.tar.gz?alt=media&token=095808fe-9364-4330-938a-5961bc828191 -->
+<!-- task: text-embedding -->
+<!-- fine-tunable: true -->
+<!-- format: saved_model_2 -->
```

Comment:
Please add a tag for  `license: custom`. The actual license can then be linked, eg. something like in this markdown (see the bottom): https://raw.githubusercontent.com/tensorflow/tfhub.dev/37fe58b699af75083d4ad9cbfb5ded3cdcbb01ce/assets/docs/silero/models/silero-stt/en/1.md